src/app/client/src/app/plugins/profile/components/delete-account/delete-account.component.ts

```diff
@@ -113,8 +114,24 @@ export class DeleteAccountComponent implements OnInit, OnDestroy {
   }
 
   verificationSuccess(data) {
-      window.location.replace('/logoff');
-      this.cacheService.removeAll();
+      this.userService.deleteUser().subscribe(
+        (data: ServerResponse) => {
+          if(_.get(data, 'result.response') === 'SUCCESS'){
+            window.location.replace('/logoff');
+            this.cacheService.removeAll();
+            if(this.deviceDetectorService.isMobile()){
+              const url ='dev.sunbird.app://mobile?userId'+ this.userProfile.userId;
+              window.open(url, '_blank');
+            }
+          }
+        },
+        (err) => {
+          console.log('------>',err);
+          const errorMessage =  this.resourceService.messages.fmsg.m0085;
+          this.toasterService.error(errorMessage);
```

Comment:
We should show the user specific error mesage.

ask @pooja for the right message

Reply:
I will add it once the correct error will be given